dev-site-config/site.config.js

```diff
@@ -1,8 +1,8 @@
-
 const siteConfig = {
   appConfig: {
     themes: {
       'Default Theme': '',
+      'Clinical Low Light Theme': 'clinical-lowlight-theme',
```

Comment:
If teams want to use lowlight, will they need to put clinical-lowlight-theme into their ThemeManager?  How will this work when a page needs to pull in both clinical-lowlight-theme and ti-clinical-lowlight-theme so that new components and old components are all colored using the lowlight theme?  Can the ThemeManager take in 2 theme names?

Reply:
No, theme provide does not support adding two theme classes, nor for most cases should it.

You've got a couple of options, you can double up theme providers or you can append the theme name to document yourself. Doubling up theme providers would look something like this.

```jsx
<ApplicationBase locale={locale} themeName="clinical-lowlight-theme" themeIsGlobal>
  <ThemeProvider themeName="ti-clinical-lowlight-theme" isGlobalTheme style={{ height: '100%' }}>
    {child}
  </ThemeProvider>
</ApplicationBase>
```